### PR TITLE
ingested amta 2000

### PR DESCRIPTION
**Pull request by davidstap**
- included page numbers
- wanted to include abstracts, but I do not think Springer allows this, what do you think @mjpost ?

Maybe it is better to use the naming convention of AMTA 2020, i.e. using Volume 1 / Volume 2 / ... for different tracks:
* Proceedings of the 14th Conference of the Association for Machine Translation in the Americas (Volume 1: Research Track) 16 papers
* Proceedings of the 14th Conference of the Association for Machine Translation in the Americas (Volume 2: User Track) 29 papers
?

Also, note that I included the pre-conference workshop as _Proceedings of the Workshop on Machine translation in practice: from old guard to new guard_, even though this was **published separately from the proceedings of the main conference**. Let me know if you agree.

### CODE REVIEW

File: data/xml/2000.amta.xml [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+      <url hash="32cce533">2000.amta-workshop.5</url>
+    </paper>
+    <paper id="6">
+      <title>Automated translation for the deployment of dynamic and mission critical content [abstract only]</title>
+      <author><first>Bob</first><last>Sheng</last></author>
+      <url hash="a24b99f0">2000.amta-workshop.6</url>
+    </paper>
+    <paper id="7">
+      <title>What do we need from <fixed-case>MT</fixed-case>? [no transcript available]</title>
+    </paper>
```
Comment (mjpost) [inline data/xml/2000.amta.xml:295]: Should be able to remove "[no transcript available]"

File: data/xml/2000.amta.xml [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+    <paper id="6">
+      <title><fixed-case>MT</fixed-case>ranslatability</title>
+      <author><first>Arendse</first><last>Bernth</last></author>
+      <author><first>Claudia</first><last>Gdaniec</last></author>
+    </paper>
+  </volume>
+  <volume id="papers" ingest-date="2021-05-09">
+    <meta>
+      <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: Technical Papers</booktitle>
+      <publisher/>
```
Comment (mjpost) [inline data/xml/2000.amta.xml:42]: Same with publisher here (maybe edit the spreadsheet and re-import)

File: data/xml/2000.amta.xml [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+      <title>Text meaning representation as a basis for representation of text interpretation</title>
+      <author><first>Stephen</first><last>Helmreich</last></author>
+      <author><first>David</first><last>Farwell</last></author>
+      <pages>179-188</pages>
+    </paper>
+  </volume>
+  <volume id="systems" ingest-date="2021-05-09">
+    <meta>
+      <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: System Descriptions</booktitle>
+      <publisher/>
```
Comment (mjpost) [inline data/xml/2000.amta.xml:171]: publisher

File: data/xml/2000.amta.xml [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+      <author><first>Kyu Baek</first><last>Hwang</last></author>
+      <author><first>Min O</first><last>Jang</last></author>
+      <author><first>Yung Taek</first><last>Kim</last></author>
+      <pages>248-251</pages>
+    </paper>
+  </volume>
+  <volume id="workshop" ingest-date="2021-05-09">
+    <meta>
+      <booktitle>Proceedings of the Workshop on Machine translation in practice: from old guard to new guard</booktitle>
+      <publisher/>
```
Comment (mjpost) [inline data/xml/2000.amta.xml:257]: publisher

**Review by mjpost — COMMENTED** [review]
Let's add the abstracts. I think we can consider them as metadata and therefore public information, in contrast to the paper itself.

Note, too, that we should add `<url>` tags with complete URLs to the Springer pages (the paper page is fine, rather than directly to the PDF). This will take a bit of time I think, but is only necessary for these few volumes that were published by Springer.

**Comment by davidstap** [comment]
okay, should be fixed by now.

File: data/xml/2000.amta.xml [hunk 5]
```diff
@@ -1,10 +1,10 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <collection id="2000.amta">
-  <volume id="tutorials" ingest-date="2021-05-09">
+  <volume id="tutorials" ingest-date="2021-05-14">
     <meta>
       <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: Tutorial Descriptions</booktitle>
-      <publisher/>
-      <address>Cuernavaca, Mexico</address>
+      <publisher>Springer</publisher>
+      <address>Cuernavaca. Mexico</address>
```
Comment (mjpost) [inline data/xml/2000.amta.xml]: Should be a comma here

File: data/xml/2000.amta.xml [hunk 6]
```diff
@@ -36,11 +36,11 @@
...
     </paper>
   </volume>
-  <volume id="papers" ingest-date="2021-05-09">
+  <volume id="papers" ingest-date="2021-05-14">
     <meta>
       <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: Technical Papers</booktitle>
-      <publisher/>
-      <address>Cuernavaca, Mexico</address>
+      <publisher>Springer</publisher>
+      <address>Cuernavaca. Mexico</address>
```
Comment (mjpost) [inline data/xml/2000.amta.xml]: comma

File: data/xml/2000.amta.xml [hunk 7]
```diff
@@ -178,49 +214,63 @@
...
+      <abstract>This paper describes some of the features of the new 32-bit Windows version of PAHO’s English-Spanish (ENGSPAN®) and Spanish-English (SPANAM®) machine translation software. The new dictionary update interface is designed to help users add their own terminology to the lexicon and encourage them to write context-sensitive rules to improve the quality of the output. Expanded search capabilities provide instant access to related source and target entries, expressions, and rules. A live system demonstration will accompany this presentation.</abstract>
     </paper>
   </volume>
-  <volume id="studies" ingest-date="2021-05-09">
+  <volume id="studies" ingest-date="2021-05-14">
     <meta>
       <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: User Studies</booktitle>
-      <publisher/>
-      <address>Cuernavaca, Mexico</address>
+      <publisher>Springer</publisher>
```
Comment (mjpost) [inline data/xml/2000.amta.xml:272]: comma

File: data/xml/2000.amta.xml [hunk 8]
```diff
@@ -157,19 +189,23 @@
...
     </paper>
   </volume>
-  <volume id="systems" ingest-date="2021-05-09">
+  <volume id="systems" ingest-date="2021-05-14">
     <meta>
       <booktitle>Proceedings of the Fourth Conference of the Association for Machine Translation in the Americas: System Descriptions</booktitle>
-      <publisher/>
-      <address>Cuernavaca, Mexico</address>
+      <publisher>Springer</publisher>
+      <address>Cuernavaca. Mexico</address>
```
Comment (mjpost) [inline data/xml/2000.amta.xml]: comma

**Review by mjpost — APPROVED** [review]
This looks fantastic. I spot-checked a few links and they seem to go to the right place. Note the small issue with periods in the `<publisher>` lines.

**Comment by mjpost** [comment]
@davidstap I think how you handled the pre-conference workshop is great. I'm not sure about the naming convention, but think I would stick with what you have here.

Maybe others in the area want to comment: @danielgildea @davidweichiang. FYI to Dan, David (Chiang): @davidstap ([web](https://davidstap.github.io)) is undertaking the task of ingesting all the old data from the MT Archive. He'll be creating PRs for the conferences one by one, in case you want to help edit. The MT Archive is down. Mikel is trying to salvage the domain, but in the meantime [we have an MT Archive archive here](https://aclanthology.org/www.mt-archive.info/). David will be working through conferences in roughly this order:

* AMTA
* MT Summit
* TMI
* TC (translating and the computer)
* IWSLT
* Int'l workshop on free / open-source MT
* Int'l workshop on example-based MT
* CLAW: was a workshop, ingest with associated conferences
* Pre-1980 conferences

**Comment by mjpost** [comment]
(It's too bad we don't have the slides or other material from [the tutorials at this conference](https://aclanthology.org/previews/ingest-amta-2000/volumes/2000.amta-tutorials/)!)

**Comment by mjpost** [comment]
@davidstap not sure why the builds are failing. Try running `make check` locally which runs our rather-strict checks on formatting. If there are errors you can usually run `make autofix` and commit the changes.

**Comment by davidstap** [comment]
@mjpost both `jing -c data/xml/schema.rnc data/xml/2000.amta.xml` and `make check` show no errors. (I also tried `make autofix`, which does not fix anything, since there seems nothing to be fixed.)

Not sure how to proceed without wasting a lot of time. Do you have any ideas?

**Comment by mjpost** [comment]
I wonder if they were just having problems yesterday? Maybe push a dummy change and see if things are okay now?

**Comment by davidstap** [comment]
Seems to work now :-)

**Comment by mjpost** [comment]
Thanks for merging, @davidstap! This should be written down somewhere, but please do a squash-merge next time—this keeps the commit history a bit cleaner and more readable. If you haven't seen it before, it's an option if you click on the expansion arrow, as pictured here, and will become the default thereafter.

image

Here are some notes on crafting the commit message:
* I usually delete all commit messages that are subsumed by the top-level message. So if the top-level message is "Ingesting AMTA 2002", I'd remove notes that document steps along the way (e.g., "added abstracts", "ran black", etc)
* If there are unrelated commits, I'll retain them as separate bullet points (e.g., "fixed bug in ingestion script").

[Here's an example](https://github.com/acl-org/acl-anthology/commit/e8f90e6953529c209a59bb3b9faa6babd8dd1349) where we retain many partial commits so as to have a detailed log of what was done, and to close related issues.

[Here's an ingestion example](https://github.com/acl-org/acl-anthology/commit/3cfec07ae80888887d078c76b2a963108b1d0183) with a single top-level merge and mention of an unrelated fix.

**Pull request merged**